rundeckapp/grails-spa/packages/ui-trellis/src/app/components/readme-motd/EditProjectFile.vue

```diff
@@ -45,25 +45,30 @@
                   </details>
                 </div>
                 <ace-editor
+                  v-if="authAdmin"
```

Comment:
Important, the behaviour of the components shouldn't change while writing tests, therefore please undo the changes on lines 48 and 68. 

When the user doesn't have permissions to create/edit files, the component will render strings (whose logic is defined inside the block that starts with `<template v-if="displayConfig.includes('none')">`) and the tests should confirm that the right strings are being rendered in such cases.